src/main/java/seedu/address/ui/cards/LineNumber.java

```diff
@@ -1,5 +1,8 @@
 package seedu.address.ui.cards;
 
+/**
```

Comment:
Perhaps you could change it to Represents a class...

Reply:
resolved in the latest push.